R/recall_data.R

```diff
@@ -129,7 +129,7 @@ recall_by_make <- function(make, manufacturer = FALSE,
 #' @examples
 #' \dontrun{
 #' recall_by_model("civic")
-#' recall_by_model(c("Subaru", "Toyota"), start_year = 2008, partial = FALSE)
+#' recall_by_model(c("ranger", "brz"), start_year = 2008, partial = TRUE)
 #' API_KEY <- "xxxxxxxxxxx"
```

Comment:
why did the above line change? `partial = TRUE` is the default behavior so doesn't really make sense to have an example with `partial = TRUE`

Reply:
Subaru and Toyota aren't models

I changed partial back to FALSE